src/main/kotlin/kotlinx/kover/api/KoverExtension.kt

```diff
@@ -6,33 +6,39 @@ package kotlinx.kover.api
 
 import org.gradle.api.model.*
 import org.gradle.api.provider.*
+import org.gradle.api.tasks.*
 
 open class KoverExtension(objects: ObjectFactory) {
 
     /**
      * Specifies whether the plugin is applied to the test task and configures it to collect and generate coverage data.
      */
     public var isEnabled: Boolean = true
+        @Input get
 
     /**
      * Specifies the coverage engine to be used to collect execution data.
      */
     public val coverageEngine: Property<CoverageEngine> = objects.property(CoverageEngine::class.java)
+        @Input get
 
     /**
      * Specifies the version of Intellij-coverage dependency.
      */
     val intellijEngineVersion: Property<String> = objects.property(String::class.java)
+        @Input get
 
     /**
      * Specifies the version of JaCoCo dependency.
      */
     val jacocoEngineVersion: Property<String> = objects.property(String::class.java)
+        @Input get
 
     /**
      * Specifies whether the reports will be generated within 'check' task execution.
      */
     val generateReportOnCheck: Property<Boolean> = objects.property(Boolean::class.java)
+        @Input get
```

Comment:
here and everywhere: it's enough to mark the field with the `@Input`/`@Output`/`@Nested` without the need to introduce getter separately, isn't it?

Reply:
Gradle searches for annotations only on getters.
Documentation to the annotation: `... Annotations on setters or just the field in Java are ignored.`

Reply:
Changed to `@get:Nested` etc